concent_api/gatekeeper/tests/test_gatekeeper_views.py

```diff
@@ -87,6 +90,30 @@ def test_upload_should_return_401_if_message_token_deadline_pass(self):
         self.assertEqual(response.status_code, 401)
         self.assertIn('message', response.json().keys())
 
+    @freeze_time("2018-12-30 11:00:00")
+    def test_upload_should_return_401_if_file_paths_are_not_unique(self):
+        self.upload_token.files.append(FileTransferToken.FileInfo())
+        self.upload_token.files[1]['path']      = 'blender/benchmark/test_task/scene-Helicopter-27-cycles.blend'
+        self.upload_token.files[1]['checksum']  = 'sha1:098f6bcd4621d373cade4e832627b4f6'
+        self.upload_token.files[1]['size']      = 1024
+
+        golem_upload_token = dump(self.upload_token, settings.CONCENT_PRIVATE_KEY, settings.CONCENT_PUBLIC_KEY)
+        encrypted_token = b64encode(golem_upload_token).decode()
+        response = self.client.post(
+            '{}{}'.format(
+                reverse('gatekeeper:upload'),
+                'blender/benchmark/test_task/scene-Helicopter-27-cycles.blend'
+            ),
+            HTTP_AUTHORIZATION = 'Golem ' + encrypted_token,
+            content_type = 'application/x-www-form-urlencoded',
+            HTTP_CONCENT_CLIENT_PUBLIC_KEY = self.public_key
```

Comment:
Please fix your style here.

Reply:
That style is in all previous method, so i used it, to keep convention. I should fix it ?

Reply:
Fix it. I let some of this stuff slide in previous reviews because we needed to merge it and this was a small issue but you should not base your new code on that :)